bin/o2a-run-all-unit-tests

```diff
@@ -16,9 +16,6 @@ set -euo pipefail
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 O2A_DIR="$( cd "${MY_DIR}/.." && pwd )"
 
-airflow initdb
-airflow resetdb -y
-
```

Comment:
If I am not mistaken it was necessary only for one test which was integration test not a unit one.

Reply:
It was only needed for the test that accessed the database. Is this test already removed?

Reply:
Yes, it was remove.